### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by ciraganenicole**
In this pull request, I  create a main.rb file that will serve as your console app entry-point in which we find:

- List all books.
- List all people.
- Create a person (teacher or student, not a plain Person).
- Create a book.
- Create a rental.
- List all rentals for a given person's id.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+      print 'Age: '
+      age = gets.chomp
+      print 'Name: '
+      name = gets.chomp
+      print 'Classroom:'
+      classroom = gets.chomp
+      print 'Has parent permission? [Y/N]: '
+      permission_resp = gets.chomp
+      parent_permission = permission_resp.downcase == 'y'
+
```
Comment (ShoiraTa) [inline main.rb:61]: - [ ] Here parent permission is being asked both, student and teacher, but teacher should have parent permission set to true by default. Kindly render parent persission only for student, here is an example: 

```ruby
    person =
      if person_type == "1"
        print "Has parent permission? [Y/N]: "
        # *If needed, ask for parameters for the option.
        parent_permission = gets.chomp
        parent_permission = parent_permission.downcase == "y"

        Student.new(age, name, parent_permission)
      elsif person_type == "2"
        print "Specialization: "
        # *If needed, ask for parameters for the option.
        specialization = gets.chomp

        Teacher.new(age, specialization, name)
      end

    @people << person
    puts "Person created successfully"
  end
```

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+      puts 'Your Library is empty'
+      return
+    end
+    @people.each do |person|
+      print "[#{person.class.name}] Name: #{person.name.capitalize}, ID: #{person.id}, Age: #{person.age}\n"
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
```
Comment (ShoiraTa) [inline main.rb:46]: - [ ] I am getting an error  `wrong number of arguments (given 0, expected 2..3) (ArgumentError)` when trying to create a student and a teacher: 
Screenshot 2022-06-16 at 12 36 13

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @ciraganenicole ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+    option = gets.chomp
+
+    case option
+    when '1'
+      print 'Age: '
+      age = gets.chomp
+      print 'Name: '
+      name = gets.chomp
+      print 'Classroom:'
+      classroom = gets.chomp
```
Comment (brytebee) [inline main.rb:57]: - [ ] **Required:** While I attempted to create a student, I was prompted to supply a classroom and when I input a number, the app crashes. Kindly fix this.

image

When the input is string I get past and when I entered "y" to parental permission, it crashes again.

image

File: main.rb [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+      print 'Has parent permission? [Y/N]: '
+      permission_resp = gets.chomp
+      parent_permission = permission_resp.downcase == 'y'
+
+      student = Student.new(age: age, classroom: classroom, name: name, parent_permission: parent_permission)
+      @people.push(student)
+
+      puts "Person created successfuly\n"
+    when '2'
```
Comment (brytebee) [inline main.rb:66]: - [ ] **Required:** I also tried to create a Teacher but the app crashed after all inputs are given.

image


### **_I Suggest that you:_** 
- Kindly see the [video instruction](https://www.youtube.com/watch?v=vkkgrhD6aXQ) again.
- Walk through every case with your implementation.
- If you don't want a data type to be supplied by the user, kindly validate it dynamically using conditionals to ensure your app is stable.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @ciraganenicole 👋, 

### Highlight
- No linter errors.✔️
- Good commit messages.✔️ 

Great work attempting to make the changes 👏 👏 
You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


# Suggested changes
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by t-yanick — APPROVED** [review]
Hi @ciraganenicole, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**